pom.xml

```diff
@@ -765,10 +765,6 @@
                                             <!-- Instead, please use
                                             `org.jspecify.annotations.Nullable`. -->
                                         </property>
-                                        <property name="illegalClasses" value="io.micrometer.core.lang.Nullable">
-                                            <!-- Instead, please use
-                                            `org.jspecify.annotations.Nullable`. -->
-                                        </property>
```

Comment:
This one is actually [still on the classpath](https://github.com/micrometer-metrics/micrometer/issues/6407#issuecomment-3500104771), but as it's deprecated, there's no risk of us silently introducing it.